src/asm3/dbupdates/35115.py

```diff
@@ -0,0 +1,8 @@
+from asm3.dbupdate import execute
+
+dbo = dbo
+l = dbo.locale
+
+## Add fluids to lksmedicaltype
```

Comment:
Future reference, the flake8 lint tool that we use really does not like comments starting with ## instead of a single #